packages/delegation-process/test/approveProducerDelegation.test.ts

```diff
@@ -4,7 +4,8 @@ import {
   getMockDelegationProducer,
   getMockTenant,
   getMockEService,
-} from "pagopa-interop-commons-test/index.js";
+  flushPDFMetadata,
+} from "pagopa-interop-commons-test";
```

Comment:
🧹

Reply:
removed in [4ac0d95](https://github.com/pagopa/interop-be-monorepo/pull/1214/commits/4ac0d95eaf780a928374aff4423d9acadc31f2f8)